### ISSUE TITLE
Error: The process '/usr/bin/xcrun' failed with exit code 1

### ISSUE THREAD

**Issue by paulz** [sersoft-gmbh/swift-coverage-action#12]
not sure what changed, tried adding a space to filter but step before with single framework has failed:

```

2
  with:
3
    search-paths: /tmp/ci-derived-data
4
    target-name-filter: ^Testable$
5
    output: ./.swiftcov
6
    format: txt
7
    ignore-conversion-failures: false
8
    fail-on-empty-output: false
9
Validating input
10
Setting up paths
11
Finding coverage files
12
  Found 1 profiling data file(s)...
13
Converting files
14
  Skipping AppSpecs due to target name filter...
15
  Skipping CombineReachability due to target name filter...
16
  Skipping CoreMLWrapper due to target name filter...
17
  Skipping ForcePlateSimulator due to target name filter...
18
  Skipping SimulatorCore due to target name filter...
19
  Error: Failed to convert /tmp/ci-derived-data/Build/Products/Debug/ForcePlateSimulator.app/Contents/Frameworks/SimulatorCore.framework/Versions/A/Frameworks/Testable.framework/Testable: Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
20
  Error: Failed to convert /tmp/ci-derived-data/Build/Products/Debug/ForcePlateSimulator.app/Contents/Frameworks/Testable.framework/Testable: Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
21
  Skipping SimulatorSpecs due to target name filter...
22
  Skipping Legible due to target name filter...
23
  Skipping Nimble due to target name filter...
24
  Skipping Quick due to target name filter...
25
  Skipping SnapshotTesting due to target name filter...
26
  Skipping JWTDecode due to target name filter...
27
  Skipping Legible due to target name filter...
28
  Skipping Nimble due to target name filter...
29
  Skipping Pluralize_swift due to target name filter...
30
  Skipping Pods_AppSpecs due to target name filter...
31
  Skipping Pods_CloudUpload due to target name filter...
32
  Skipping Pods_ForcePlateService due to target name filter...
33
  Skipping Pods_Simulator due to target name filter...
34
  Skipping Pods_SimulatorSpecs due to target name filter...
35
  Skipping Pods_SpartaScan due to target name filter...
36
  Skipping Pods_SpartaScan_UITests due to target name filter...
37
  Skipping Pods_UnitSpecs due to target name filter...
38
  Skipping Quick due to target name filter...
39
  Skipping Reachability due to target name filter...
40
  Skipping SDWebImage due to target name filter...
41
  Skipping SDWebImageSwiftUI due to target name filter...
42
  Skipping SimulatorCore due to target name filter...
43
  Error: Failed to convert /tmp/ci-derived-data/Build/Products/Debug/SimulatorCore.framework/Versions/A/Frameworks/Testable.framework/Testable: Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
44
  Skipping SimulatorSpecs due to target name filter...
45
  Skipping SnapshotTesting due to target name filter...
46
  Skipping Sparta Scan due to target name filter...
47
  Skipping CombineReachability due to target name filter...
48
  Skipping CoreMLWrapper due to target name filter...
49
  Skipping Lottie due to target name filter...
50
  Skipping Pluralize_swift due to target name filter...
51
  Skipping Reachability due to target name filter...
52
  Skipping SDWebImage due to target name filter...
53
  Skipping SDWebImageSwiftUI due to target name filter...
54
  Skipping Swinject due to target name filter...
55
  Skipping SwinjectAutoregistration due to target name filter...
56
  Error: Failed to convert /tmp/ci-derived-data/Build/Products/Debug/Sparta Scan.app/Contents/Frameworks/Testable.framework/Testable: Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
57
  Skipping Yams due to target name filter...
58
  Skipping AppSpecs due to target name filter...
59
  Skipping JWTDecode due to target name filter...
60
  Skipping Legible due to target name filter...
61
  Skipping Nimble due to target name filter...
62
  Skipping Quick due to target name filter...
63
  Skipping SnapshotTesting due to target name filter...
64
  Skipping ForcePlateSimulator due to target name filter...
65
  Skipping SimulatorCore due to target name filter...
66
  Error: Failed to convert /tmp/ci-derived-data/Build/Products/Debug/Sparta Scan.app/Contents/Resources/ForcePlateSimulator.app/Contents/Frameworks/SimulatorCore.framework/Versions/A/Frameworks/Testable.framework/Testable: Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
67
  Error: Failed to convert /tmp/ci-derived-data/Build/Products/Debug/Sparta Scan.app/Contents/Resources/ForcePlateSimulator.app/Contents/Frameworks/Testable.framework/Testable: Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
68
  Skipping SimulatorSpecs due to target name filter...
69
  Skipping Legible due to target name filter...
70
  Skipping Nimble due to target name filter...
71
  Skipping Quick due to target name filter...
72
  Skipping SnapshotTesting due to target name filter...
73
  Skipping Swifter due to target name filter...
74
  Skipping Swinject due to target name filter...
75
  Skipping SwinjectAutoregistration due to target name filter...
76
  Skipping UITests-Runner due to target name filter...
77
  Skipping XCTAutomationSupport due to target name filter...
78
  Skipping XCTest due to target name filter...
79
  Skipping UITests due to target name filter...
80
  Skipping CombineReachability due to target name filter...
81
  Skipping CoreMLWrapper due to target name filter...
82
  Skipping Lottie due to target name filter...
83
  Skipping Pluralize_swift due to target name filter...
84
  Skipping Reachability due to target name filter...
85
  Skipping SDWebImage due to target name filter...
86
  Skipping SDWebImageSwiftUI due to target name filter...
87
  Skipping Swifter due to target name filter...
88
  Skipping Swinject due to target name filter...
89
  Skipping SwinjectAutoregistration due to target name filter...
90
  Skipping Yams due to target name filter...
91
  Skipping UITests due to target name filter...
92
  Skipping UnitSpecs due to target name filter...
93
  Skipping Legible due to target name filter...
94
  Skipping Nimble due to target name filter...
95
  Skipping Quick due to target name filter...
96
  Skipping Yams due to target name filter...
97
  Skipping Lottie due to target name filter...
98
Error: Conversion failures:
99
Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
100
Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
101
Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
102
Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
103
Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
104
Error: The process '/usr/bin/xcrun' failed because one or more lines were written to the STDERR stream
```

using `sersoft-gmbh/swift-coverage-action@main` it's hard to understand which version of action actually ran.

_Originally posted by @paulz in https://github.com/sersoft-gmbh/swift-coverage-action/issues/11#issuecomment-889690481_

**Comment by paulz** [comment]
using v2 of the action same failure happened on the next target:

with debug enabled:
```
##[debug]Writing coverage report to /Users/sparta/actions-runner/_work/mariposa/mariposa/.swiftcov/Testable.framework.coverage.txt
51235
  ##[debug]Found match of type xctest: /tmp/ci-derived-data/Build/Products/Debug/UnitSpecs.xctest
51236
  ##[debug]Project name: UnitSpecs
51237
  Skipping UnitSpecs due to target name filter...
51238
  ##[debug]Found match of type framework: /tmp/ci-derived-data/Build/Products/Debug/UnitSpecs.xctest/Contents/Frameworks/Legible.framework
51239
  ##[debug]Project name: Legible
51240
  Skipping Legible due to target name filter...
51241
  ##[debug]Found match of type framework: /tmp/ci-derived-data/Build/Products/Debug/UnitSpecs.xctest/Contents/Frameworks/Nimble.framework
51242
  ##[debug]Project name: Nimble
51243
  Skipping Nimble due to target name filter...
51244
  ##[debug]Found match of type framework: /tmp/ci-derived-data/Build/Products/Debug/UnitSpecs.xctest/Contents/Frameworks/Quick.framework
51245
  ##[debug]Project name: Quick
51246
  Skipping Quick due to target name filter...
51247
  ##[debug]Found match of type framework: /tmp/ci-derived-data/Build/Products/Debug/Yams/Yams.framework
51248
  ##[debug]Project name: Yams
51249
  Skipping Yams due to target name filter...
51250
  ##[debug]Found match of type framework: /tmp/ci-derived-data/Build/Products/Debug/lottie-ios/Lottie.framework
51251
  ##[debug]Project name: Lottie
51252
  Skipping Lottie due to target name filter...
51253
  ::endgroup::
51254
##[debug]Enhanced Annotations not enabled on the server. The 'title', 'end_line', and 'end_column' fields are unsupported.
51255
Error: Conversion failures:
51256
Error: The process '/usr/bin/xcrun' failed with exit code 1
51257
##[debug]Node Action run completed with exit code 1
```

**Comment by paulz** [comment]
may be it's related to #11 

as the run summary shows:
```
Failed to convert /tmp/ci-derived-data/Build/Products/Debug/Sparta Scan.app/Contents/MacOS/Sparta Scan: Error: The process '/usr/bin/xcrun' failed with exit code 1
```

<img width="981" alt="image" src="https://user-images.githubusercontent.com/59230/127620998-771442a0-fadc-473b-b579-43160e29d6d7.png">

**Comment by ffried** [comment]
> may be it's related to #11

Oh, yea it totally is. The action (like Codecov before) simply appends `Contents/MacOS/$NAME` to the path where $NAME is the target name (in this case `Sparta Scan`). Which doesn't exist in this case and thus fails.

I'll fix this by checking the target path and trying again with whitespaces removed. I'll have to think of some clever way to allow users of the action to perform additional adjustments, but for now removing whitespaces seems like a common enough case to simply bake it in.

**Comment by paulz** [comment]
resolved with #11
